bitnami/elasticsearch/values-production.yaml

```diff
@@ -117,16 +125,11 @@ securityContext:
   fsGroup: 1001
   runAsUser: 1001
 
-## Elasticsearch discovery node parameters
-##
-discovery:
-  name: discovery
-
 ## Elasticsearch coordinating-only node parameters
 ##
 coordinating:
   name: coordinating-only
-  replicas: 2
+  replicas: 3
```

Comment:
Why this change?

Reply:
We are increasing the number of nodes from 2 to 3 in the case of `master` and `data`, why not for `coordinating` and `ingest`?